### DISCUSSION TITLE
Rhino/Grasshopper - abaques connection

### DISCUSSION THREAD

**Discussion by chienHH** [BHoM/BHoM#1406] [General]
hi there,

I would like to know if there are any near future plans or possibilities to connect rhino/grasshopper with dynamic FEA softwares such as Abaques?

Thanks for your effort!

regards,
Chien

**Comment by alelom** [comment]
Hi @chienHH , I believe there had been discussion of adding an Abaqus Toolkit but demand has never been enough, I think. Always happy to support people wanting to contribute and code it though! :)

**Reply by alelom** [reply]
Also check the other FEA software that are currently supported, such as Robot and SAP
